spec/controllers/api/v1/courses_controller_spec.rb

```diff
@@ -23,5 +23,30 @@ module V1
         end
       end
     end
+
+
+    describe UsersController do
+      describe 'create' do
+        it 'throws a 501 error' do
+          post :create
+          expect(response.status).to eq(501)
+        end
+      end
+
+      describe 'index' do
+        it 'throws a 501 error' do
+          get :index
+          expect(response.status).to eq(501)
+        end
+      end
+
+      describe 'destroy' do
+        it 'throws a 501 error' do
+          delete :destroy, params: { id: 16 }
+          expect(response.status).to eq(501)
+        end
+      end
+    end
+
```

Comment:
Since these controllers exist in the same module, this test actually doesn't test the nested routes (that I think you were going for) but rather the routes at the `api/v1/courses`.  In order to test those, we would need a separate controller.  I think a good way to think about this is with the following example URIs:

`GET api/v1/users/{users_id}`
`GET api/v1/courses/{course_id}/users/{users_id}`

If both of these routes reference the same controller and method `controllers/api/v1/users_controller#read`, then they will do the exact same thing no matter what.  But what if a user isn't in a specific course.  They should still be able to be processed using the first endpoint but they shouldn't be able to be processed using the second endpoint for the course they are not in (it should 404 or something instead).